nemo/collections/llm/recipes/CONFIGURATION-HIERARCHY.md

```diff
@@ -20,27 +20,27 @@
   <blockquote>
 
   ```sh
-  tensor_model_parallel_size: int = 1
-  pipeline_model_parallel_size: int = 1
-  virtual_pipeline_model_parallel_size: Optional[int] = None
-  context_parallel_size: int = 1
-  sequence_parallel: bool = False
-  expert_model_parallel_size: int = 1
-  pipeline_dtype: Optional[torch.dtype] = None,
+  tensor_model_parallel_size: int = 1 # Intra-layer model parallelism. Splits tensors across GPU ranks
```

Comment:
should these be added as docstrings to the code instead?

Reply:
these comments are in code too. I added them here so that a user can consult this map and get all necessary info in one place, instead going back and forth to search each module that are in multiple places in NeMo repo and megatron-lm repo.